src/main/resources/swagger/api-docs.yaml

```diff
@@ -3343,6 +3343,34 @@ paths:
         500:
           description: Internal Error
 
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/importPFB:
+    post:
+      x-passthrough: false
+      tags:
+        - Entities
+      operationId: importPFB
+      summary: Import PFB data from an [Avro](https://avro.apache.org/) file
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/workspaceNamespaceParam'
+        - $ref: '#/parameters/workspaceNameParam'
+        - in: body
+          description: JSON object containing PFB URL
+          name: pfbImportRequest
+          required: true
+          schema:
+            $ref: '#/definitions/PFBRequest'
+      responses:
+        204:
+          description: Successful Request
+        400:
+          description: Bad Request
+        404:
+          description: Workspace not found
```

Comment:
if Arrow returns a 404 - I assume it might, if the PFB url is bad - then this API will also return a 404 and "Workspace not found" is not correct.

Reply:
Actually, returns a 500 with the message from Arrow (which could be 404 or, in the case of accessing something that doesn't exist in a Google bucket, 403).

Reply:
Unfortunately, the way I wrote the code, I'm pretty sure it's going to do the same thing with a 404 from Rawls, so... I guess I need to handle at least some errors differently... maybe all.

I guess this is a good reason for those tests that you asked about.